src/main/java/frc/robot/subsystems/LEDSubsystem.java

```diff
@@ -58,6 +58,14 @@ public void rainbow() {
     m_rainbowFirstPixelHue %= 180;
   }
 
+  public void rainbowFlash() {
+    for (int i = 0; i < m_ledBuffer.getLength(); i++) {
+      m_ledBuffer.setHSV(i, m_rainbowFirstPixelHue, 255, 128);
+    }
+    m_rainbowFirstPixelHue += 1;
+    m_rainbowFirstPixelHue %= 180;
+  }
+
```

Comment:
not in the scope of this pr - remove

Reply:
f
can I add it to the led branch instead?